readme.md

```diff
@@ -0,0 +1,17 @@
+## How to install Fabric3 on Red Hat distribution ##
+
+1. First, be sure that Python 3.4+ is installed on your computer.
+2. Run "sudo python3 -m pip install Fabric3"
```

Comment:
Github markdown flavour also includes "code" blocks. See [this](https://guides.github.com/features/mastering-markdown/)